recipe/meta.yaml

```diff
@@ -25,82 +25,104 @@ source:
-    - m2-git    # [win]
-  host:
-    # For some reason cmake and ninja need to be installed
-    # alongside python in the host
-    # https://github.com/conda-forge/pytorch-cpu-feedstock/pull/21#discussion_r541397252
-    - cmake
-    - ninja
-    - git       # [not win]
-    - m2-git    # [win]
-    # GPU requirements
-    #- cudatoolkit {{ cudatoolkit }}*  # [pytorch_variant == "gpu"]
-    #- cudnn {{ cudnn }}*              # [pytorch_variant == "gpu"]
-    #- magma                           # [pytorch_variant == "gpu"]
-    # other requirements
-    - python
-    - numpy
-    - pip
-    - setuptools
-    - pyyaml
-    - requests
-    - future
-    - six
-    - cffi
-    - mkl-devel {{ mkl }}
-    - mkl {{ mkl }}
-    - libblas * *_mkl
-    - typing
-    - libuv       # [unix]
-    - pkg-config  # [unix]
-    #- intel-openmp  # [osx]
-  run:
-    - mkl {{ mkl }}
-    - libblas * *_mkl
-    #- intel-openmp   # [osx]
-    - llvm-openmp    # [osx]
-    #- _pytorch_select ==0.1             # [pytorch_variant == "cpu"]
-    #- _pytorch_select ==0.2             # [pytorch_variant == "gpu"]
-    # GPU requirements
-    #- {{ pin_compatible('cudatoolkit', max_pin='x.x') }}  # [pytorch_variant == "gpu"]
-    #- {{ pin_compatible('cudnn') }}                       # [pytorch_variant == "gpu"]
-    # other requirements
-    - python
-    - {{ pin_compatible('numpy') }}
-    - cffi
-    - future  # [py2k]
-    - typing  # [py2k]
+outputs:
+  - name: pytorch
+    build:
+      string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+      string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [cuda_compiler_version == "None"]
+      detect_binary_files_with_prefix: False
+    script: build_pytorch.sh  # [not win]
+    script: bld_pytorch.bat   # [win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler('cuda') }}    # [cuda_compiler_version != "None"]
+        # Dec 2020: it seems that git is broken on windows, so we use m2-git
+        - patch     # [not win]
+        - m2-patch  # [win]
+        - git       # [not win]
+        - m2-git    # [win]
+        - libgomp   # [linux]
+        - llvm-openmp    # [osx]
+      host:
+        # For some reason cmake and ninja need to be installed
+        # alongside python in the host
+        # https://github.com/conda-forge/pytorch-cpu-feedstock/pull/21#discussion_r541397252
+        - cmake
+        - git       # [not win]
+        - m2-git    # [win]
+        - ninja
+        # GPU requirements
+        - cudnn                           # [cuda_compiler_version != "None"]
+        - nccl                            # [cuda_compiler_version != "None"]
+        - magma                           # [cuda_compiler_version != "None"]
+        # other requirements
+        - python
+        - numpy
```

Comment:
Seeing this build warning:
```
WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.
Falling back to default numpy value of 1.11
```
That's probably not desired, should be 1.16?

Reply:
i think that warning is benign